src/components/messenger/list/view-group-information-panel/index.test.tsx

```diff
@@ -51,6 +62,11 @@ describe(ViewGroupInformationPanel, () => {
     expect(wrapper).toHaveElement(IconUsers1);
   });
 
+  it('renders edit button when current user is admin', () => {
```

Comment:
Probably should have the inverse test too to ensure people who can't edit don't see it.